src/constants/networks.ts

```diff
@@ -73,6 +73,10 @@ export const FALLBACK_URLS: { [key in SupportedChainId]: string[] } = {
     // "Safe" URLs
     `https://alfajores-forno.celo-testnet.org`,
   ],
+  [SupportedChainId.BOBA]: [
+    // "Safe" URLs
+    `https://lightning-replica.boba.network/`,
```

Comment:
should we use mainnet.boba.network instead? cc @InoMurko

Reply:
it's a list, let's add all! :-)

Reply:
Mainnet rpc has been added